staff/oriol-bover/fly-me/flyme-api/README.md

```diff
@@ -0,0 +1,6 @@
+# FLY ME PROJECT - SERVER
+
+## DESCRIPTION
+
+Server side of the FLY ME PROJECT. The server works using node, express and mongo. 
```

Comment:
ADD full brief describing the project, what's about, etc. an intro to any new person landing on it and requiring to understand it from just reading this simple text.